packages/plugin-interactive-tools/sources/commands/upgrade-interactive.tsx

```diff
@@ -106,23 +118,23 @@ export default class UpgradeInteractiveCommand extends BaseCommand {
         : descriptor.range;
 
       const [resolution, dependency] = await Promise.all([
-        fetchUpdatedDescriptor(descriptor, descriptor.range, referenceRange),
-        fetchUpdatedDescriptor(descriptor, descriptor.range, `latest`),
+        fetchUpdatedDescriptor(descriptor, descriptor.range, referenceRange).catch(() => null),
+        fetchUpdatedDescriptor(descriptor, descriptor.range, `latest`).catch(() => null),
```

Comment:
I was getting some errors even in the existing view. When only one component throws an error then the rest of the list is able to recover, but when it's the main component catching it then it caused the command to hang. 

Obviously, just silencing them isn't great, but is there a way of bubbling up error messages for non-fatal errors?

Reply:
Indeed I remember seeing errors corrupt the display 🤔 I think this will be fine for now

Reply:
Just for future reference, I was getting this error:

```
(node:41169) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open '/Users/ross/Projects/personal/berry/.patches/@strictsoftware/typedoc-plugin-monorepo.patch'
(node:41169) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:41169) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```